packages/gatsby/src/commands/develop.ts

```diff
@@ -168,6 +150,26 @@ module.exports = async (program: IProgram): Promise<void> => {
     })
   }
 
+  // NOTE(@mxstbr): We need to start the develop proxy before the develop process to ensure
+  // codesandbox detects the right port to expose by default
+  const proxy = startDevelopProxy({
+    proxyPort: proxyPort,
+    targetPort: developPort,
+    program,
+  })
+
+  const developProcess = new ControllableScript(`
+    const cmd = require(${JSON.stringify(developProcessPath)});
+    const args = ${JSON.stringify({
+      ...program,
+      port: developPort,
+      proxyPort,
+      // Don't pass SSL options down to the develop process, it should always use HTTP
+      ssl: null,
```

Comment:
can you explain how this fixes ssl? haha, it _looks_ like it does the opposite 😆

Reply:
I think the child process needs to be HTTP regardless whereas the parent one (proxy) is SSL